src/styles/card.scss

```diff
@@ -0,0 +1,95 @@
+@import './variables';
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+
+  &__title {
+    color: $color-main;
```

Comment:
Ensure that the variable `$color-main` is defined in the imported `variables` file. If it's not defined, this will cause a compilation error.